symphony/app/fbcnms-projects/hub/app/components/workflow/pages/workflowList/Scheduling/Scheduling.js

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from "react";
+import {
+  Accordion,
+  Button,
+  Card,
+  Col,
+  Container,
+  Row,
+  Table
+} from "react-bootstrap";
+import { withRouter } from "react-router-dom";
+import PageCount from "../../../common/PageCount";
+import PageSelect from "../../../common/PageSelect";
+import { conductorApiUrlPrefix, frontendUrlPrefix } from "../../../constants";
+import SchedulingModal from "./SchedulingModal/SchedulingModal";
+import superagent from "superagent";
+
+const Scheduling = props => {
+  const [showSchedulingModal, setShowSchedulingModal] = useState(false);
+  const [activeRow, setActiveRow] = useState();
+
+  const refresh = () => {
+    const path = conductorApiUrlPrefix + "/schedule/";
+    const req = superagent.get(path).accept("application/json");
+    req.end((err, res) => {
+      if (res && res.ok && Array.isArray(res.body)) {
+        const result = res.body;
+        let size = Math.floor(result.length / defaultPages);
+        let dataset = result.sort((a, b) =>
+            a.name > b.name ? 1 : b.name > a.name ? -1 : 0
+          ) || [];
+        setData(dataset);
+        setPagesCount(result.length % defaultPages ? ++size : size);
+        deselectActiveRow();
+      } else {
+        // TODO
+      }
+    });
+    return [];
+  };
+
+  const [data, setData] = useState(refresh);
+  const [pagesCount, setPagesCount] = useState(1);
+  const [defaultPages, setDefaultPages] = useState(20);
+  const [viewedPage, setViewedPage] = useState(1);
```

Comment:
Could these go on line 21 considering that `refresh()`, which is above these, references them?